tracer/build/_build/Build.GitHub.cs

```diff
@@ -55,6 +57,7 @@ partial class Build
 
     Target AssignPullRequestToMilestone => _ => _
        .Unlisted()
+       .Requires(() => GitHubRepositoryName)
```

Comment:
I think we also need to add this to the following targets?

- GenerateReleaseNotes 
- CompareCodeCoverageReports 
- CompareBenchmarksResults

Reply:
Good point.
Added in 185a33d11b66f283490412e023dc10701378ae36 along with all target, where `GitHubRepositoryName` is used.